tests/test_flow_integration.py

```diff
@@ -98,9 +103,17 @@ async def test_large_repository(request: pytest.FixtureRequest) -> None:
         "token": "",
     }
 
-    response = client.post("/", data=form_data)
-    assert response.status_code == status.HTTP_200_OK, f"Request failed: {response.text}"
-    assert "Mocked Template Response" in response.text
+    response = client.post("/api/ingest", data=form_data)
+    # This might fail with 400 if repo doesn't exist, or succeed with 200
+    _valid = {status.HTTP_200_OK, status.HTTP_400_BAD_REQUEST}
+    assert response.status_code in _valid, f"Request failed: {response.text}"
```

Comment:
I don't get this: `# This might fail with 400 if repo doesn't exist, or succeed with 200`. We know that the repo here does not exist (`https://github.com/large/repo-with-many-files`).

```suggestion
    assert response.status_code in status.HTTP_400_BAD_REQUEST, f"Request did not fail: {response.text}"
```